samwise.js

```diff
@@ -5,24 +5,22 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?